sql/explain.go

```diff
@@ -48,9 +51,20 @@ func (p *planner) Explain(n *parser.Explain) (planNode, error) {
 	if err != nil {
 		return nil, err
 	}
-	plan, err = markExplain(plan, mode)
-	if err != nil {
-		return nil, fmt.Errorf("%v: %s", err, n)
+	switch mode {
+	case explainDebug:
+		plan, err = markExplain(plan, mode)
+		if err != nil {
+			return nil, fmt.Errorf("%v: %s", err, n)
+		}
+		return plan, nil
+	case explainPlan:
+		v := &valuesNode{}
+		v.columns = []string{"Level", "Type", "Description"}
+		explainNode(v, plan, 0)
+		plan = v
+	default:
+		return nil, fmt.Errorf("unsupported EXPLAIN mode: %d", mode)
```

Comment:
Yes, but only because we exit earlier if `mode == explainNone`. Seems safer to leave this in in case a new mode gets added without enhancing this switch.